aptos-move/move-examples/marketplace/sources/coin_listing.move

```diff
@@ -39,6 +39,8 @@ module coin_listing {
     const ENOT_SELLER: u64 = 6;
 
     // Core data structures
+    const FIXED_PRICE_TYPE: u8 = 1;
+    const AUCTION_TYPE: u8 = 2;
```

Comment:
tbh this is better as a string. Having an int enum is confused, it's also prone to errors.

Reply:
Sure, this does have a non-negligible overhead though.